sdk/src/state_machine/core/mocks.rs

```diff
@@ -48,8 +52,35 @@ impl ZkVM for MockZkvm {
     fn verify(
         _proof: Self::Proof,
         _code_commitment: &Self::CodeCommitment,
-    ) -> Result<<<Self as ZkVM>::Proof as crate::zk::traits::ProofTrait<Self>>::Output, Self::Error>
+    ) -> Result<<<Self as ZkVm>::Proof as crate::zk::traits::ProofTrait<Self>>::Output, Self::Error>
     {
         todo!()
     }
 }
+
+#[derive(Default)]
+pub struct MockWitness {
```

Comment:
Should we move out of mocks? Can we make it a default witness?

Reply:
Done